spec/collection-spec.coffee

```diff
@@ -117,12 +132,12 @@ describe 'Brainstem.Collection', ->
 
     context 'options has a name property', ->
       it 'uses options name property over the collections brainstemKey', ->
-        loadObjectSpy = spyOn(base.data, 'loadObject').andReturn(new $.Deferred)
+        loadObjectSpy = spyOn(storageManager, 'loadObject').and.returnValue(new $.Deferred)
```

Comment:
Since we're using Jasmine 2, we can take advantage of `beforeAll` to clean up a lot of the repetition we have in the spec setup here.

Probably something we can do later since I understand this is just a straight port of what we had before.